dashboard/api.py

```diff
@@ -246,14 +246,19 @@ def get_info_for_course(course, mmtrack):
 
     def _add_run(run, mmtrack_, status):
         """Helper function to add a course run to the status dictionary"""
-        course_data['runs'].append(
-            format_courserun_for_dashboard(
-                run,
-                status,
-                mmtrack=mmtrack_,
-                position=len(course_data['runs']) + 1
-            )
+        formatted_run = format_courserun_for_dashboard(
+            run,
+            status,
+            mmtrack=mmtrack_,
+            position=len(course_data['runs']) + 1
         )
+        if run.is_current and formatted_run['has_paid']:
```

Comment:
travis ci is crashing on  add`mmtrack_.is_enrolled` check. Any idea @annagav @pdpinch